app/components/project-card-skills.js

```diff
@@ -1,21 +1,73 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  set,
+} = Ember;
+
+/**
+  `project-card-skills` composes the given project's list of skills on the
+  project's card.
+
+  ## default usage
+
+  ```Handlebars
+  {{project-card-skills skills=project.skills}}
+  ```
+
+  @class project-card-skills
+  @module Component
+  @extends Ember.Component
+ */
+export default Component.extend({
   classNames: ['project-card-skills'],
+
+  /**
+    Returns whether or not the overflow skills are hidden.
```

Comment:
It's been a while since we've worked on this, so I don't really remember what the idea was.

Do we want the same end-result as with project members, as in

* Show a total of X skills on the card
* If there are more than X skills total, show an indicator that some are hidden

If so, as far as I can remember, this is implemented differently due to two reasons

* We are showing only the skills the user also possesses (an overlap of project skills and user skills)
* Skills, unlike project members, are labels of variable length, not icons, so we can't really maintain a simple visual consistency by showing the same number of skills on each card.

We should probably look into making the behavior similar, to project members, for clarity's sake, but I don't think we can make it too similar.

What I think would simplify things a bit, or at least make them clearer in the project/user overlap part is merging of #466. At that point, a project has project-skills and a user has user-skills, so we overlap those two by skill id, then fetch the skills for it, instead of going with the `hasSkill` route.

What to do about variable label length, though, I'm really not sure. If we can somehow constrain the labels to all be the same size, it would make things simpler, but it would also make the list look less appealing. Any ideas here are welcome.